docs/get-started/table-theme-premade.qmd

```diff
@@ -37,21 +41,49 @@ Below are the first two premade styles. The first uses `color="blue"`, and the s
 :::{.g-col-lg-6 .g-col-12}
 
 ```{python}
-gt_ex.opt_stylize(style = 1, color = "blue")
+gt_ex.opt_stylize(style=1, color="blue")
 ```
 
 :::
 :::{.g-col-lg-6 .g-col-12}
 ```{python}
-gt_ex.opt_stylize(style = 2, color = "red")
+gt_ex.opt_stylize(style=2, color="red")
 ```
 :::
 ::::::
 
-Notice that first table (blue) emphasizes the row labels with a solid background color. The second table (red) emphasizes the column labels, and uses solid lines to separate the body cell values.
+Notice that first table (blue) emphasizes the row labels with a solid background color. The second table (red) emphasizes the column labels, and uses solid lines to separate the body cell values. There are six options for the `color=` argument: `"blue"`, `"cyan"`, `"pink"`, `"green"`, `"red"`, and `"gray"`.
```

Comment:
Can we replace this sentence enumerating all colors with a link to the reference page (which enumerates them)? could add a note w/ a link like "see `.opt_stylize()` for all color options"

Maybe we could also add a link on the `.opt_stylize()` page to this guide? (e.g. "for examples of each style, see {Linked Name of Guide Page}")

Reply:
Sounds good! I'll make those changes.